tests/blackhole/test_silicon_driver_bh.cpp

```diff
@@ -229,10 +229,9 @@ TEST(SiliconDriverBH, UnalignedStaticTLB_RW) {
                 // Statically mapping a 1MB TLB to this core, starting from address NCRISC_FIRMWARE_BASE.  
                 device.configure_tlb(i, core, get_static_tlb_index_callback(core), l1_mem::address_map::NCRISC_FIRMWARE_BASE);
             }
+            device.setup_core_to_tlb_map(i, get_static_tlb_index_callback);
         } 
     }
-
-    device.setup_core_to_tlb_map(get_static_tlb_index_callback);
```

Comment:
How did this compile?

Reply:
It didn't compile before, we don't check for blackhole :) Actually I can add a build for blackhole, even though we don't have blackhole cards to run on.